content/pt-br/docs/reference/glossary/storage-class.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Classe de Armazenamento
+id: storageclass
+date: 2018-04-12
+full_link: /docs/concepts/storage/storage-classes
+short_description: >
+  Uma Classe de Armazenamento oferece uma maneira para os administradores descreverem diferentes tipos de armazenamento disponíveis.
+
+aka: 
+tags:
+- core-object
+- storage
+---
+ Uma classe de Armazenamento oferece uma maneira para os administradores descreverem diferentes tipos de armazenamento disponíveis.
+
+<!--more--> 
+
+
+Classes de Armazenamento podem mapear para niveis de qualidade de serviço, políticas de backup, ou políticas arbitrárias determinadas por administradores do cluster. Cada Classe de Armazenamento contêm os campos `provisionador`, `parametros`, e `políta de recuperação`, que são usados quando um {{< glossary_tooltip text="Volume Persistente" term_id="persistent-volume" >}} pertencente à classe precisa ser provisionada dinamicamente. Usuários podem solicitar uma classe específica usando o nome de um objeto Classe de Armazenamento.
```

Comment:
Just a small thing here:
The fields (campos) can't be translated, so you can leave it in english.
`provisioner, parameters, e reclaimPolicy`

Reply:
@edsoncelio 
Thanks for catching it! Changed!